inst/stan/epinowcast.stan

```diff
@@ -163,48 +175,92 @@ model {
       rd_eff_sd ~ zero_truncated_normal(rd_eff_sd_p[1], rd_eff_sd_p[2]);
     } 
   }
+  // share of cases with eventually known reference date
+  alpha_sd ~ normal(alpha_sd_p[1], alpha_sd_p[2]) T[0, ];
+  for (k in 1:g){
+    alpha_start[k] ~ normal(alpha_start_p[1], alpha_start_p[2]);
+    alpha_epsilon[k] ~ normal(0, alpha_sd);
+  }
   // reporting overdispersion (1/sqrt)
   sqrt_phi ~ normal(sqrt_phi_p[1], sqrt_phi_p[2]) T[0,];
   }
   // log density: observed vs model
   if (likelihood) {
     profile("model_likelihood") {
-    target += reduce_sum(obs_lupmf, st, 1, flat_obs, sl, csl, imp_obs, sg, st,
-                         rdlurd, srdlh, ref_lh, dpmfs, ref_p, phi);
+    target += obs_lupmf(flat_obs | obs_miss, dmax, sl, csl, g, imp_obs, sg, st,
+                         rdlurd, srdlh, ref_lh, dpmfs, ref_p, alpha, phi);
     }
   }
 }
 
 generated quantities {
   int pp_obs[pp ? sum(sl) : 0];
+  int pp_obs_miss[pp ? sum(sl) : 0];
   vector[ologlik ? s : 0] log_lik;
+  vector[ologlik ? (t-dmax) : 0] log_lik_miss;
   int pp_inf_obs[cast ? dmax : 0, cast ? g : 0];
+  int pp_inf_obs_miss[cast ? t : 0, cast ? g : 0];
+  int pp_inf_obs_miss_rep[cast ? (t-dmax) : 0, cast ? g : 0];
```

Comment:
As discussed, we here restrict the reporting date - related quantities to the days after the maximum delay (starting at dmax + 1).